rdma_link_linux.go

```diff
@@ -143,3 +143,128 @@ func (h *Handle) RdmaLinkSetName(link *RdmaLink, name string) error {
 
 	return execRdmaSetLink(req)
 }
+
+func netnsModeToString(mode uint8) string {
+	netnsModes := map[uint8]string{0: "exclusive", 1: "shared"}
```

Comment:
What is the point of instantiating a map for the time span of this function for such a simple logic?
It would make sense if this map is used somewhere else, and therefore defined at package level. 
Otherwise following will do it:
```
func netnsModeToString(mode uint8) string {
  switch(mode) {
  case 0:
     return "exclusive"
  case 1:
     return "shared"
  default:
     return "unknown"
  }
}
```

Reply:
@aboch done.